esm/interface/document-fragment.js

```diff
@@ -8,4 +9,15 @@ export class DocumentFragment extends NonElementParentNode {
   constructor(ownerDocument) {
     super(ownerDocument, '#document-fragment', DOCUMENT_FRAGMENT_NODE);
   }
+
+  get textContent() {
+    const text = [];
+    let {[NEXT]: next, [END]: end} = this;
+    while (next !== end) {
+      if (next.nodeType === TEXT_NODE)
+        text.push(next.textContent);
```

Comment:
not sure if `node.data` is what you want but I think should do it too

Reply:
`node.data`?

what i want is to fix https://github.com/WebReflection/linkedom/issues/231

Reply:
I meant `next.data` which is a direct getter ... `textContent` passes through `data` so is a tiny bit slower https://github.com/WebReflection/linkedom/blob/main/esm/interface/character-data.js#L41-L51

Reply:
How do I reconciliate with this?

![image](https://github.com/WebReflection/linkedom/assets/74560659/3fea622a-b0ef-4fc2-a63e-cf49f0e91ed3)

Reply:
data is a standard CharacterData accessor and it's the fastest one out there for CharacterData instances, namely `#text` and `#comment` ... you can see the implementation in the CharacterData class: https://developer.mozilla.org/en-US/docs/Web/API/CharacterData/data

all other things are **not CharacterData** ... you are screening per `TEXT_NODE` and that **is a CharacterData**.

Use the right tool for the job 😉